src/ui/components/CircleGraph.jsx

```diff
@@ -0,0 +1,41 @@
+
+// eslint-disable-next-line no-unused-vars
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const CircleGraph = ({ number, text }) => {
+  const circleStyle = {
+    width: '200px',
+    height: '200px',
+    borderRadius: '50%',
+    backgroundColor: '#F6BA27', 
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'center',
+    alignItems: 'center',
+    color: '#fff',
+    fontSize: '75px', // Tamaño del número
+    textAlign: 'center',
+  };
+
+  const textStyle = {
+    fontSize: '18px', // Tamaño del texto
+  };
+
+  return (
+    <div style={circleStyle}>
+      <div>{number}</div>
+      <div style={textStyle}>{text}</div>
+    </div>
+  );
+};
+
+CircleGraph.propTypes = {
```

Comment:
Since we are going to use 2 equal components add as props 4 atributes that will be topLeftText, topRightText, bottomLeftText, bottomRightText and a prop por each corner where we can insert the endpoints. For topLeftText, there will be a topLeftApiURL